### PR TITLE
Implement notification preview mentioned in issue #1235

### PR DESCRIPTION
**Pull request by kunzef**
## Issue 

This fixes the following issue:
- Resolves: #1235
- Helps detect further issues with notifications

## Screenshot
Notification Overview          |  Opening
:-------------------------:|:-------------------------:
WhatsApp Image 2023-05-25 at 00 46 48|screen-20230525-004136

## Why this is useful for all students
As the App has experienced some problems with the notifications it may be useful to check scheduled notifications for completeness.

## Description Design
The overview is started by clicking five consecuitve time at the Debug Information Textview. After two clicks a Toast informing that the Notification Overview will open after more clicks is displayed.

## Description functionality
This view tries to displays all notifications that are scheduled or active. It uses a `StickyListHeadersListView` as in the LectureFragment to divide the displayed notifications in parts sorted according to way they were retrieved. 
The sources are:
- currently active notifications
- all elements in the database _scheduled_notifications_
- all elements in the databse _active_alarms_
- PendingIntents for every NotificationType
- PendingIntents for every element in _scheduled_notifications_

### CODE REVIEW

**Comment by COM8** [comment]
I suggest you still add this feature/view to all branches(master) and hide it behind some kind of "Debug Option". This Debug option could be enabled via for example pressing 5 times on the app version inside the settings/about page. Perhaps clean it up a bit before e.g. adding margins so it looks a bit more in line with the usual UI.

**Comment by kunzef** [comment]
Hey all, I just marked this as ready for review as I believe it can be added to the project. I updated the description of the PR to reflect the current state. I would love to get some feedback.

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -309,8 +309,19 @@
     <string name="send_email">E-Mail Feedback senden</string>
     <string name="show_feedback">Feedback senden</string>
     <string name="feedbackSubj">Feedback für TUM Campus App</string>
-    <!-- END Translated, please do not change identifiers -->
+    <string name="show_notification_view_progress_single">Die Benachrichtigungsübersicht öffnet sich mit 1 Klick.</string>
+    <string name="show_notification_view_progress_multiple">Die Benachrichtigungsübersicht öffnet sich mit %1$d Klicks.</string>
```
Comment (CommanderStorm) [inline app/src/main/res/values-de/strings.xml:313]:
```suggestion
    <string name="show_notification_view_progress_single">Die Benachrichtigungsübersicht öffnet sich in einem Klick.</string>
    <string name="show_notification_view_progress_multiple">Die Benachrichtigungsübersicht öffnet sich in %1$d Klicks.</string>
```

**Review by CommanderStorm — APPROVED** [review]
Once the formatting checks pass, this will be automerged

**Pull request merged**